### PR TITLE
AWS RDS - adding postgresql dashboard and collection instruction

### PR DESCRIPTION
**Pull request by sumoanema**
## PLEASE READ

Following a recent back-end update, all contributors with a local clone or fork of our repository are required to run `yarn install`. This does not apply to [direct page edits](https://help.sumologic.com/docs/contributing/edit-doc/#minor-edits).

- [ ] Yes, I've run `yarn install`
- [X] No, does not apply to me

## Purpose of this pull request

This pull request is to add information around the newly added dashboard to the existing AWS RDS app. These dashboard are developed around postgresql in AWS RDS.

JIRA: 
https://sumologic.atlassian.net/browse/SUMO-234809

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions, .clabot
- [X] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

**Review by JV0812 — APPROVED** [review]
LGTM

File: docs/integrations/amazon-aws/rds.md [hunk 1]
```diff
@@ -148,6 +149,18 @@ The Amazon RDS app uses the following logs and metrics:
    "logGroup":"/aws/rds/instance/rds-dbinstance-1/audit"
 }
 ```
+
+```json title ="Authentication Failure (Postgresql Log)"
```
Comment (kimsauce) [inline docs/integrations/amazon-aws/rds.md:153]:
```suggestion
```json title ="Authentication Failure (PostgreSQL Log)"
```

File: docs/integrations/amazon-aws/rds.md [hunk 2]
```diff
@@ -148,6 +149,18 @@ The Amazon RDS app uses the following logs and metrics:
...
+{"timestamp":1705859815000,
+"message":"2024-01-21 17:56:55 UTC:3.92.54.14(57164):postgresql@postgresql:[43627]:FATAL:  password authentication failed for user \"postgresql\""
+}
+```
+
+```json title ="Slow Queries (Postgresql Log)"
```
Comment (kimsauce) [inline docs/integrations/amazon-aws/rds.md:159]:
```suggestion
```json title ="Slow Queries (PostgreSQL Log)"
```

File: docs/integrations/amazon-aws/rds.md [hunk 3]
```diff
@@ -546,9 +593,79 @@ Use this dashboard to:
...
+* Obtain recent and top fatal and error events.
+* Obtain recent queries running into error with error message.
+
+<img src={useBaseUrl('img/integrations/amazon-aws/Amazon-RDS-PostgreSQL-Logs-Errors.png')} alt="Amazon RDS dashboard" />
+
+### 14. Amazon RDS - PostgreSQL Logs - Slow Query Overview
+
+The **Amazon RDS - PostgreSQL Logs - Slow Query Overview** dashboard provides an overview of the slow query logs. AWS RDS will report slow logs with statement taking more than threshold value given through `log_min_duration_statement`. This dashboard can be filtered with different values for query execution time through `slow_query_latency_ms`.
+
+Use this dashboard to :
```
Comment (kimsauce) [inline docs/integrations/amazon-aws/rds.md:631]:
```suggestion
Use this dashboard to: 
```

File: docs/integrations/amazon-aws/rds.md [hunk 4]
```diff
@@ -546,9 +593,79 @@ Use this dashboard to:
...
+* Obtain unique slow queries along with execution time, analysing minimum, maximum, average, and many more.
+* Obtain time comparison between number of slow queries and their execution time over 1 day or 1 week.
+  
+<img src={useBaseUrl('img/integrations/amazon-aws/Amazon-RDS-PostgreSQL-Logs-Slow-Query-Overview.png')} alt="Amazon RDS dashboard" />
+
+### 15. Amazon RDS - PostgreSQL Logs - Slow Query Details
+
+The **Amazon RDS - PostgreSQL Logs - Slow Query Details** dashboard provides details on slow log query. Also, this dashboards displays the distribution of slow queries along with parameters like database and query type.
+
+Use this dashboard to :
```
Comment (kimsauce) [inline docs/integrations/amazon-aws/rds.md:644]:
```suggestion
Use this dashboard to: 
```

File: docs/integrations/amazon-aws/rds.md [hunk 5]
```diff
@@ -546,9 +593,79 @@ Use this dashboard to:
...
+* Obtain the frequently fired slow queries.
+* Monitor the recent DML, DDL, and TCL statement which lead to slow queries.
+
+<img src={useBaseUrl('img/integrations/amazon-aws/Amazon-RDS-PostgreSQL-Logs-Slow-Query-Details.png')} alt="Amazon RDS dashboard" />
+
+### 16. Amazon RDS - PostgreSQL Logs - Security
+
+The **Amazon RDS - PostgreSQL Logs - Security** dashboard provides details with respect to login failures and threat intel along with activity by default user.
+
+Use this dashboard to :
```
Comment (kimsauce) [inline docs/integrations/amazon-aws/rds.md:655]:
```suggestion
Use this dashboard to: 
```

File: docs/integrations/amazon-aws/rds.md [hunk 6]
```diff
@@ -546,9 +593,79 @@ Use this dashboard to:
...
+* Monitor database shut down and system up events.
+* Identify default user's authentication and generic activities.
+
+<img src={useBaseUrl('img/integrations/amazon-aws/Amazon-RDS-PostgreSQL-Logs-Security.png')} alt="Amazon RDS dashboard" />
+
+### 17. Amazon RDS - PostgreSQL Logs - Query Execution Time
+
+The **Amazon RDS - PostgreSQL Logs - Query Execution Time** dashboard provides details around the time its taking to execute queries on your PostgreSQL instance. 
+
+Use this dashboard to :
```
Comment (kimsauce) [inline docs/integrations/amazon-aws/rds.md:667]:
```suggestion
Use this dashboard to: 
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
formatting

**Review by kimsauce — APPROVED** [review]

**Pull request merged**